package.json

```diff
@@ -44,6 +44,7 @@
         "electron-is-dev": "^1.2.0",
         "electron-localshortcut": "^3.2.1",
         "electron-log": "^4.3.1",
+        "electron-media-service": "github:tidal-engineering/electron-media-service",
```

Comment:
Please move this dependency into `optionalDependencies` segment, since it's only used in macOS currently. Thanks!